Assets/Scripts/CardCounter.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class CardCounter : MonoBehaviour
+{
+
+    public TextMeshProUGUI cardCounter;
+    private int cardsLeft;
+    private MemMad memMad;
+
+    // Start is called before the first frame update
+    void Start()
```

Comment:
```suggestion
    private void Start()
```
I believe this and Update() can be made private